distributed-commandbus-jgroups/src/main/java/org/axonframework/jgroups/commandhandling/JGroupsConnector.java

```diff
@@ -186,13 +255,19 @@ public void connect() throws Exception {
         SimpleMember<Address> localMember = new SimpleMember<>(localName, localAddress, LOCAL_MEMBER, null);
         members.put(localAddress, new VersionedMember(localMember, membershipVersion.getAndIncrement()));
         updateConsistentHash(ch -> ch.with(localMember, loadFactor, commandFilter));
+        if (!executorProvided) {
+            executorService = Executors.newCachedThreadPool(new AxonThreadFactory("JGroupsConnector"));
```

Comment:
Maybe add localName to the thread names? Makes it easier to debug when logs from multiple instances are merged into the same logging system

Reply:
Smart! Added `localName` to the `AxonThreadFactory`.